lms/templates/fields/field_dropdown.underscore

```diff
@@ -17,11 +23,11 @@
             <option value="<%- selectOption[0] %>"><%- selectOption[1] %></option>
         <% }); %>
     </select>
-    <a href="#" class="u-field-value-display">
-        <span class="sr"><%- screenReaderTitle %></span>
+    <button class="u-field-value-display">
+        <span class="sr"><%- screenReaderTitle %> &nbsp;</span>
```

Comment:
yes we need to have two instances of it in the field. As the visual text would only appear in the ```u-field-value-readonly``` span. The remaining spans are just description for the screen reader.